xml/System.Reflection.Emit/AssemblyBuilder.xml

```diff
@@ -2683,7 +2683,11 @@ The following code example shows how to define and use a dynamic assembly. The e
       </ReturnValue>
       <Docs>
         <summary>Gets the host context where the dynamic assembly is being created.</summary>
-        <value>A value that indicates the host context where the dynamic assembly is being created.</value>
+        <value>
+          .NET Framework only: A value that indicates the host context where the dynamic assembly is being created.
+
+          .NET Core and .NET 5+: 0 in all cases.
```

Comment:
Nit: `AssemblyBuilder.HostContext` is also a virtual user overridable property in current .NET.